src/onegov/election_day/import_export/mappings.py

```diff
@@ -0,0 +1,279 @@
+"""
+
+Following mappings should be used to map csv headers and database columns
+for import and export.
+
+Keys are the columns from the CSV Files, values are col names returned by
+database query.
+
+For complex queries, it is much better to write them in pure sql. Return
+the table as named tuples and work with that and this mapping.
+
+"""
+
+
+#  --- INTERNAL
+
+INTERNAL_COMMON_ELECTION_HEADERS = [
```

Comment:
You are inconsistent in this file with lists/tuples. I think you should always use tuples. You can still combine tuples like this:
```python
foo = (1, 2)
bar = (*foo, 3, 4)
```

Reply:
thanks! i'll do that...